app/assets/javascripts/admin/controllers/admin-users-list-show.js.es6

```diff
@@ -40,13 +45,23 @@ export default Ember.Controller.extend({
   }, 250).observes('listFilter'),
 
   _refreshUsers: function() {
-    var self = this;
     this.set('refreshing', true);
 
-    AdminUser.findAll(this.get('query'), { filter: this.get('listFilter'), show_emails: this.get('showEmails') }).then(function (result) {
-      self.set('model', result);
-    }).finally(function() {
-      self.set('refreshing', false);
+    AdminUser.findAll( { filter: this.get('listFilter'), show_emails: this.get('showEmails') }).then( (result) => {
+      this.set('model', result);
+    }).finally( () => {
+      this.set('refreshing', false);
+    });
+  },
+
+  @observes('order', 'ascending')
+  sortUsers: function() {
```

Comment:
This function seems to be a duplication of `_refreshUsers` above. Is there a reason why can't pass the order into `_refreshUser` as well? :thinking: